src/bar.scss

```diff
@@ -23,8 +23,8 @@ $block: #{$fd-namespace}-bar;
     padding: 0 0.5rem;
     margin-top: 0.5rem;
     margin-bottom: 0.5rem;
-    margin-left: $margin-x;
-    margin-right: $margin-x;
+    left: $margin-x;
+    right: $margin-x;
```

Comment:
Floating bar examples are not working well cause of changes in bar.scss
![image](https://user-images.githubusercontent.com/26483208/93207787-c6589480-f75b-11ea-9989-2d6519afe959.png)
Let's add some let's add left: 0 / right: 0 by default

Reply:
Fixed! Thanks for catching it